imap_processing/lo/l2/lo_l2.py

```diff
@@ -38,9 +41,7 @@ def lo_l2(sci_dependencies: dict, anc_dependencies: list) -> list[xr.Dataset]:
         psets = sci_dependencies["imap_lo_l1c_pset"]
 
         # Create the rectangular sky map from the pointing set.
-        lo_rect_map = project_pset_to_rect_map(
-            psets, spacing_deg=6, spice_frame=geometry.SpiceFrame.ECLIPJ2000
-        )
+        lo_rect_map = project_pset_to_sky_map(psets, descriptor)
```

Comment:
You updated the hi variable name to remove the rect, should we also do that here so we are consistent across all of the instruments? (and elsewhere in this file)
```suggestion
        output_map = project_pset_to_sky_map(psets, descriptor)
```